src/main/java/data/Timeline.java

```diff
@@ -0,0 +1,33 @@
+package data;
+
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * Created by Bart.
+ * Abstract class for timelines.
+ */
+public abstract class Timeline {
+
+    // Description of this timeline
+    @Getter
+    private String description;
+
+    // Lowest count of this timeline
+    @Getter @Setter
+    private int beginCount;
```

Comment:
Is there not another way to keep track of time? We need to couple this is real time later on in the project, is this the smartest way to do this? I can imagine tracking time coupled events would be easier with time oriented classes. What do you think about this?